### PR TITLE
Portfolio: setup and Mobile version skeleton

### PR DESCRIPTION
**Pull request by TekyRey**
### **Setup and Mobile version skeleton**
In this module i:

- Created a nav section
- added a background image
- added an introduction section
- added a social media links section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+</head>
+<body>
+<main>
+     <section class="phone-screen">
+        
+  <header class="header">
+    <nav>
+        <div class="logo">
+            <p><a href="#">My Logo</a></p>
+        </div>
```
Comment (omar-muhamad) [inline index.html]: - [ ] Nice work wrapping the site logo inside an `<a>` tag but kindly for semantic reasons the `<div>` and `<p>` tags and leave the `<a>` tag directly in the `<nav>`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    
+        <div class="menu-icon">
+            <button type="button"><img src="images/menu-icon.svg" alt="" aria-hidden="true"></button>   
+        </div>
```
Comment (omar-muhamad) [inline index.html]: - [ ] Same here kindly put the `<button>` tag directly on the nav and remove this wrapping `div`.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <div class="introduction">
+                <h1>I'm Rehema <br> Glad to see you</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+        
+            <div class="connect">
+                <p>let's connect</p>
+              <div class="social-platforms">
+                <a href="#"><img class="social-image" src="images/social-media-icons.png" alt="social media icons" ></a>
+              </div>
```
Comment (omar-muhamad) [inline index.html]: - [ ] For the social icons Please use a `<ul>` tag to group them, `<li>` per icon, and each icon has a `<a>` link tag.
for example: 
```
<ul>
   <li>
       <a href="your soical link path here">put the Icon here</a>
   </li>
    ....................
    .....................
</ul>
```

File: style.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  place-items: center;
+}
```
Comment (omar-muhamad) [inline style.css]: - [ ] For header styling Kindly make the hamburger button aligned beside the logo but on the right like the given Figma design.
- [ ] Please add the correct margin-top value for the header by decreasing it a little bit. and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @TekyRey,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by topeogunleye — APPROVED** [review]
### Hi @TekyRey, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**